packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -200,17 +203,17 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
   ///
   /// This could be set to false (hiding the name) if the value's description
   /// will make the name self-evident.
-  bool get showName => getBooleanMember('showName', true);
+  bool? get showName => getBooleanMember('showName', true);
 
   /// Description to show if the node has no displayed properties or children.
-  String getEmptyBodyDescription() => getStringMember('emptyBodyDescription');
+  String? getEmptyBodyDescription() => getStringMember('emptyBodyDescription');
 
   /// Hint for how the node should be displayed.
   DiagnosticsTreeStyle get style {
     return _style ??= getStyleMember('style', DiagnosticsTreeStyle.sparse);
   }
 
-  DiagnosticsTreeStyle _style;
+  DiagnosticsTreeStyle? _style;
```

Comment:
use late initialization for this instead of a private / getter / setter combination

Reply:
done